### ISSUE TITLE
[discussion] Add normalization option that takes area into account

### ISSUE THREAD

**Issue by davidebbo** [davidbau/covid-19-chart#49]
Some context on Twitter: https://twitter.com/fearthecowboy/status/1248622602257489920

I'm not sure how interesting this is, but it's worth thinking about. i.e. chart the number of cases per sq km. Or maybe some fancy formula that somehow takes both area and population into account.

Thinking aloud, I'd expect that for a given population, the situation would get worse as the area gets smaller, as people are intrinsically closer together. e.g. to take extremes, DC and Alaska have similar populations but massive size difference. Of course, things get tricky because large territories tend to have their population in big clusters. e.g. more than half of Alaska's population in in Anchorage metro, while extremely large areas are completely unpopulated (and sort of irrelevant).

Anyway, let's discuss whether this is a direction that might make sense.

**Comment by davidbau** [comment]
I guess we could report infections per square mile, but that's the opposite of the proportionality that you would expect.

What might be interesting is a scatterplot of localities rate of infection growth versus density.  The number of human interactions that a single person has in a day seems like it would be proportional to the density of population in a locality, and you might guess that this would also be proportional to the rate of spread of the infection.

So you'd expect density and rate-of-spread to show up as a line.

Localities that fall off the line would suggest that there is policy or behavior change that is making the population behave similarly to a more or less dense community.

**Comment by davidbau** [comment]
In other words, if you look at this graph, the question is, are the slopes predicted by population density?

https://covid19chart.org/#/?series=deaths&norm=first&start=%3E%3D100

**Comment by davidebbo** [comment]
The scatter plot you suggest would be for a point in time (for each locality), and not something displayed over time, right? I think doing this at the county level might be interesting, because counties are small and the population density is somewhat consistent. At the 'Alaska' level, it feels like it would become more random do to the extreme variance in density through the locality.

PS: I had been wondering what `norm=first` meant, and I just understood it in context of this :)

**Comment by davidbau** [comment]
>
> The scatter plot you suggest would be for a point in time (for each
> locality), and not something displayed over time, right? I think doing this
> at the county level might be interesting, because counties are small and
> the population density is somewhat consistent.
>
Right.   Actually each locality would lie on a fixed point on the x axis,
and at a fixed the point in time, each locality would lie on a single point
on the y axis.

Maybe if visualized over time it could be seen as an animation.

**Comment by davidebbo** [comment]
> Actually each locality would lie on a fixed point on the x axis, and at a fixed the point in time, each locality would lie on a single point on the y axis.

Yep, makes sense.

Adding animations to have an additional 'axis' would be interesting for various scenarios.

I'll leave this density idea alone for now, as it's unclear whether it would produce something meaningful to the viewer.
